extensions/wikia/Wall/WallThread.class.php

```diff
@@ -77,31 +81,31 @@ private function loadReplyObjs() {
 		}
 	}
 
-	private function loadReplyIdsFromDB($master = false) {
+	/**
+	 * Fetches reply IDs for the thread, using a limit to control large queries
+	 *
+	 * @param DatabaseBase $dbr Database resource
+	 * @param integer $afterId The last reply ID after which the next set is selected
+	 * @return array List of reply IDs
+	 */
+	private function getReplyIdsFromDB( $dbr, $afterId = null ) {
```

Comment:
According to @mklucsarits note it'll be very rarely the case that we use parameter: https://github.com/Wikia/app/pull/7012/files#r29190827
We always have to find right balance between performance optimisations vs code readability. In this case both ways are fine with me.